src/main/webapp/resources/js/components/tabs/TabPaneContent.jsx

```diff
@@ -0,0 +1,37 @@
+/**
+ * Component to render proper layout for a tab.
+ * Use this to display content in a tab pane.
+ */
+
+import React from "react";
+import PropTypes from "prop-types";
+import { Col, PageHeader } from "antd";
+
+import { SPACE_MD } from "../../styles/spacing";
+
+/**
+ * Stateless UI component for creating vertical tabs
+ * @param {string} title - title for tab
+ * @param {object} children - content to display in tab
+ *
+ * @returns {Element} - Returns a component which displays the tab title and content
+ */
+
+export function TabPaneContent({ title, children }) {
+  return (
+    <Col span={12}>
+      <PageHeader
+        style={{ padding: 0, paddingBottom: SPACE_MD }}
+        title={title}
```

Comment:
Let's add the `...props` here and document it.  I want this because then you can use all of the `PageHeader` attributes.